packages/app-page-builder/src/blockEditor/config/eventActions/saveBlock/getPreviewImage.ts

```diff
@@ -23,10 +24,14 @@ function getDataURLImageDimensions(dataURL: string): Promise<ImageDimensionsType
     });
 }
 
-interface createdImageType {
-    data: any;
+interface CreatedImageType {
+    data: File | {};
 }
-export default async function getPreviewImage(element: any, meta: any): Promise<createdImageType> {
+export default async function getPreviewImage(
```

Comment:
Please avoid default exports. Export named functions, components, etc. We're trying to get rid of all default exports in the system over time, as they introduce various issues; from module system compatibility, to poor "Find Usages" in IDEs.